client/app/styles/rjsf/ArrayFieldTemplate.tsx

```diff
@@ -67,8 +68,9 @@ const ArrayFieldTemplate = ({
           </div>
         );
       })}
-      {canAdd && (
+      {canAdd && !disabled && (
```

Comment:
Should we only hide this when disabled for internal users or is it fine to hide for everyone.

Reply:
Unknown, @nanyangpro ?

Reply:
Talked with Nan, will hide (rather than disable) button for everyone